app/models/target.rb

```diff
@@ -0,0 +1,6 @@
+class Target < ApplicationRecord
+  belongs_to :topic
+  belongs_to :user
+
+  validates_presence_of :title, :radius, :latitude, :longitude, :topic_id
```

Comment:
```suggestion
  validates :title, :radius, :latitude, :longitude, :topic_id, presence: true
```
Take a look at [this](https://guides.rubyonrails.org/active_record_basics.html#validations).